### PR TITLE
validate LURRY

### PR DESCRIPTION
**Pull request by anonWeb3Devv**
# Validate [LURRY](https://solscan.io/token/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/lurrytheduck/status/1783124340386513015
- Coingecko/ CMC URL (If available): Still working to get coingecko set up for LURRY

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
- [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by viktoravelino** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/2nhjjqSkA8FYCUdJvQhYjbtZdPjZbNo8VtNKTkJ3hncb.png?size=lg&key=78937d
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Review by viktoravelino — COMMENTED** [review]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by thiagosanrocha** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by rodrigoaraujolima92** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by iourivolkov** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by julianotavares** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by adrielborges** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by aleilson** [comment]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

File: validated-tokens.csv [hunk 1]
```diff
@@ -1422,3 +1422,4 @@ wifejak,WIFE,4y3oUrsJfSp431R3wJrWiaLxRPsnYtpkVJmoV2bYpBiy,6,https://wifejak.nyc3
 alux jownes,JOWNES,8m9fjYycXAFva1kScttQgsESVZT7yELhjZASqfHBuMa5,6,https://bafkreig7kpsty6rokwswaysqfjq576eell335236tzztmpv6p536mvfida.ipfs.nftstorage.link,true
 Catcoin,CAT,7hWcHohzwtLddDUG81H2PkWq6KEkMtSDNkYXsso18Fy3,3,https://raw.githubusercontent.com/catcoincom/Audit/mastar/logo1.png,true
 GUMMY,GUMMY,H7bTHGb5Cvo5fGe5jBDNDPUv8KykQnzyZA3qZ8sH7yxw,9,https://bafkreih3pz2wklsnoae5zuyfvascdm3phyffimvyxbsi34rjnqzkjldopu.ipfs.nftstorage.link,true
+LURRY,LURRY,6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ,9,https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791,true
```
Comment (marcosvnunes) [inline validated-tokens.csv:1425]: - [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Review by marcosvnunes — COMMENTED** [review]
- [x] Mint address: 6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ
- [x] Symbol: LURRY
- [x] Token Decimals: 9
- [x] Token icon: https://dd.dexscreener.com/ds-data/tokens/solana/6DEbzgDNQYkgk7sFZ1cTBjeUK9gpsLjgURY9EtHkpyCJ.png?size=lg&key=045791
- [x] On-chain metadata matches
- [x] No duplication on Jupiter's strict list

**Comment by Jymfletzfur** [comment]
insufficient volume

**Pull request closed without merging**